runtime/test/test-helper.js

```diff
@@ -0,0 +1,135 @@
+/**
+ * @license
+ * Copyright (c) 2017 Google Inc. All rights reserved.
+ * This code may only be used under the BSD style license found at
+ * http://polymer.github.io/LICENSE.txt
+ * Code distributed by Google as part of this project is also
+ * subject to an additional IP rights grant found at
+ * http://polymer.github.io/PATENTS.txt
+ */
+'use strict';
+
+import {assert} from './chai-web.js';
+import Arc from '../arc.js';
+import Manifest from '../manifest.js';
+import Loader from '../loader.js';
+import Planner from '../planner.js';
+import MockSlotComposer from './mock-slot-composer.js';
+
+class TestHelper {
```

Comment:
Add brief class header documentation on intent, maybe something briefly about the complexity of slot composer, what specific areas the helper is structured to make easier to test, sample usage (though there's enough that maybe sample usage is too variable to usefully illustrate in a doc comment and people can just be expected to look at the test(s) that use it).

Reply:
added comments